modules/network/vpc/README.md

```diff
@@ -1,20 +1,28 @@
 ## Description
 
-This module creates a new VPC network along with a
-[cloud NAT](https://github.com/terraform-google-modules/terraform-google-cloud-nat),
-[Router](https://github.com/terraform-google-modules/terraform-google-cloud-router)
-and common [firewall rules](https://github.com/terraform-google-modules/terraform-google-network/tree/master/modules/firewall-rules).
-This module is based on submodules defined by the
-[Cloud Foundation Toolkit](https://cloud.google.com/foundation-toolkit).
-
-The created cloud NAT (Network Address Translation) allows virtual machines
-without external IP addresses to create outbound connections to the internet.
-For more information see the [docs](https://cloud.google.com/nat/docs/overview).
-
-The following firewall rules are created with the VPC network:
-
-* Allow SSH access from the Cloud Console ("35.235.240.0/20").
-* Allow traffic between nodes within the VPC
+This module creates a new [VPC network][vpc], subnetworks, and [Cloud
+Router][router]. By default, it will enable
```

Comment:
nit: This list reads odd to me, I think because you use the verb `enable` to introduce the list and then all items also use `enable`. Maybe if you s/it will enable/it will create/.